### DISCUSSION TITLE
docs/configuration

### DISCUSSION THREAD

**Discussion by giscus** [mikro-orm/mikro-orm#5335] [Comments]
# docs/configuration

Entity Discovery

https://mikro-orm.io/docs/configuration

**Comment by MrHIDEn** [comment]
Hi 
1. How to set connection timeout  in the config for MongoDriver?
I need to adjust  timeout.
2. How to catch ECONNREFUSED with nestjs, or any way?

**Reply by B4nan** [reply]
You can use `driverOptions` to pass any additional options down to the mongo driver.

https://mikro-orm.io/docs/configuration#driver

**Comment by kastrovisk** [comment]


**Reply by B4nan** [reply]
If you need help, you need to provide a reproduction.

**Comment by bpn1604** [comment]
Hi I need to connect two databse with onstance on mikro orm , suggest a way to it
